features/src/main/feature/camel-features.xml

```diff
@@ -827,35 +827,35 @@
         <feature prerequisite='true'>wrap</feature>
         <feature version='[2.16,2.17)'>jackson</feature>
         <feature version="[4,5)">http-client</feature>
-        <bundle dependency='true'>wrap:mvn:org.apache.httpcomponents/httpasyncclient/${httpasyncclient-version}</bundle>
-        <bundle dependency='true'>wrap:mvn:co.elastic.clients/elasticsearch-java/8.12.1</bundle>
-        <bundle dependency='true'>wrap:mvn:org.elasticsearch.client/elasticsearch-rest-client/8.12.1</bundle>
-        <bundle dependency='true'>wrap:mvn:org.elasticsearch.client/elasticsearch-rest-client-sniffer/8.12.1</bundle>
+        <bundle dependency='true'>wrap:mvn:org.apache.httpcomponents/httpasyncclient/${httpasyncclient-version}$Bundle-Version=${httpasyncclient-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:co.elastic.clients/elasticsearch-java/8.12.1$Bundle-Version=8.12.1</bundle>
+        <bundle dependency='true'>wrap:mvn:org.elasticsearch.client/elasticsearch-rest-client/8.12.1$Bundle-Version=8.12.1</bundle>
+        <bundle dependency='true'>wrap:mvn:org.elasticsearch.client/elasticsearch-rest-client-sniffer/8.12.1$Bundle-Version=8.12.1</bundle>
```

Comment:
we are supposed to use `elasticsearch-java-client-sniffer-version`

Reply:
Done 👍